bedrock/firefox/tests/test_base.py

```diff
@@ -398,7 +398,7 @@ def test_fx_72_0_0_fallback(self, render_mock):
         req.locale = 'es-ES'
         self.view(req, version='72.0')
         template = render_mock.call_args[0][1]
-        assert template == ['firefox/whatsnew/index.html']
+        assert template == ['firefox/whatsnew/index-account.html']
```

Comment:
I don't see a change that would cause this test to change. Is it currently failing?

Reply:
Yeah this is currently failing on master. The branch that merged wasn't up to date, so the failure didn't get picked up by Circle.

Reply:
That is an issue with our current process. Maybe something we can fix next year.